app/celery/service_callback_tasks.py

```diff
@@ -124,13 +127,30 @@ def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
 def _send_data_to_service_callback_api(self, data, service_callback_url, token, function_name):
     object_id = data["notification_id"] if "notification_id" in data else data["id"]
     try:
-        response = requests_session.request(
-            method="POST",
-            url=service_callback_url,
-            data=json.dumps(data),
-            headers={"Content-Type": "application/json", "Authorization": f"Bearer {token}"},
-            timeout=5,
-        )
+        request_kwargs = {
+            "method": "POST",
+            "url": service_callback_url,
+            "data": json.dumps(data),
+            "headers": {"Content-Type": "application/json", "Authorization": f"Bearer {token}"},
+            "timeout": 5
+        }
```

Comment:
keys in quotes or not? from what i see this is the standard but is there a difference?